### ISSUE TITLE
error on bonus_star creation in scroller tutorial

### ISSUE THREAD

**Issue by maacl** [defold/editor2-issues#2961]
### Expected behaviour
Game object created 
### Actual behaviour
Object created but with error message:

java.lang.ClassCastException: Unknown
### Steps to reproduce
Create bonus star game object as per tutorial
<hr/>

<table>
<tr><td>Build time</td><td>2020-04-28T07:54:34.552027</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>39749b921d8e37b9a669f4096a3faa6d7252dec5</td></tr>
<tr><td>Defold engine sha</td><td>e22f6d2f81e7c53ebcbfefe703ff22ce5da252c0</td></tr>
<tr><td>Defold version</td><td>1.2.168</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/organizations/defold/issues/?query=id%3A"4d59adb49bd744b6a0ea5df44d34c69b"'>4d59adb49bd744b6a0ea5df44d34c69b</a></td></tr>
<tr><td>GPU</td><td>Mesa Intel(R) UHD Graphics 630 (CFL GT2)</td></tr>
<tr><td>GPU Driver</td><td>OpenGL ES-CM 1.1 Mesa 20.0.4</td></tr>
<tr><td>Java version</td><td>11.0.1+13</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>OS name</td><td>Linux</td></tr>
<tr><td>OS version</td><td>5.4.0-29-generic</td></tr>
<table>

**Comment by eangelin** [comment]
Similar error: https://github.com/defold/editor2-issues/issues/2959

**Comment by maacl** [comment]
Is further information needed to progress this? It would appear a serious problem that defold is unusable on the latest release of the most popular GNU/Linux platform.

**Comment by bagucode** [comment]
Did you check this? https://defold.com/faq/faq/#linux

**Comment by maacl** [comment]
Yes as mentioned in the issue I did try the workaround to no effect.
